### DISCUSSION TITLE
Portable version

### DISCUSSION THREAD

**Discussion by EliorMachlev** [visioncortex/vtracer#60] [Q&A]
Is there a portable version of this software, in which I don't need to install anything separately (like rust of Phyton) as it comes pre-packaged?

**Comment by tyt2y3** [comment]
There are some binaries in https://github.com/visioncortex/vtracer/releases/tag/0.6.0

**Comment by IIIIIIIllllllllIIIII** [comment]
> There are some binaries in https://github.com/visioncortex/vtracer/releases/tag/0.6.0

that may sound stupid but how do i open the mac binaries? they have no extension... neither .zip not .app works :)
